packages/docs/components/CodeSnippet/CodeSnippet.tsx

```diff
@@ -56,7 +56,10 @@ export const CodeSnippet: React.FC<EditorProps> = (props) => {
         disabled
         language={language}
         // Adds background and text color
-        style={editorTheme.plain}
+        style={{
+          ...editorTheme.plain,
+          overflow: 'auto',
```

Comment:
Looks like something broke mobile:
![Screenshot 2024-01-22 at 13 27 08](https://github.com/bigcommerce/big-design/assets/10539418/0a4ca990-c783-41a7-801c-381671493c72)

This should fix that.